website/docs/r/failure_domain.html.markdown

```diff
@@ -0,0 +1,55 @@
+---
+subcategory: "Fabric"
+layout: "nsxt"
+page_title: "NSXT: nsxt_failure_domain"
+description: A resource to configure a Failure Domain.
+---
+
+# nsxt_failure_domain
+
+This resource provides a method for the management of a Failure Domain.
+
+This resource is supported with NSX 3.0.0 onwards.
+
+## Example Usage
+
+```hcl
+data "nsxt_failure_domain" "failure_domain" {
+  display_name            = "failuredomain1"
+  description             = "failuredomain"
+  preferred_edge_services = "active"
+  tag {
+    scope = "scope1"
+    tag   = "tag1"
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `display_name` - (Required) Display name of the resource.
+* `description` - (Optional) Description of the resource.
+* `preferred_edge_services` - Set preference for failure domain. Set preference for edge transport node failure domain which will be considered while doing auto placement of logical router, DHCP and MDProxy on edge node.
+  `active`: For preemptive failover mode, active edge cluster member allocation prefers this failure domain.
```

Comment:
Lets put those values in same line as general description, for sake of consistency. New line with offset generally describes nested attributes rather than enumeration values.

Reply:
Thanks. Done.